hops/featurestore.py

```diff
@@ -1885,17 +1887,144 @@ def connect(host, project_name, port = 443, region_name = constants.AWS.DEFAULT_
         :project_name: the name of the project hosting the feature store to be used
         :port: the REST port of the Hopsworks cluster
         :region_name: The name of the AWS region in which the required secrets are stored
+        :secrets_store: The secrets storage to be used. Either secretsmanager, parameterstore or local
+        :api_key_file: path to a file containing an API key. For secrets_store=local only.
 
     Returns:
         None
     """
-    os.environ[constants.ENV_VARIABLES.REMOTE_ENV_VAR] = 'True'
+    dbfs_folder = "/dbfs/" + cert_folder
     os.environ[constants.ENV_VARIABLES.REST_ENDPOINT_END_VAR] = host + ':' + str(port)
     os.environ[constants.ENV_VARIABLES.HOPSWORKS_PROJECT_NAME_ENV_VAR] = project_name
     os.environ[constants.ENV_VARIABLES.REGION_NAME_ENV_VAR] = region_name
+    os.environ[constants.ENV_VARIABLES.API_KEY_ENV_VAR] = util.get_secret(secrets_store, 'api-key', api_key_file)
     project_info = rest_rpc._get_project_info(project_name)
-    os.environ[constants.ENV_VARIABLES.HOPSWORKS_PROJECT_ID_ENV_VAR] = str(project_info['projectId'])
+    project_id = str(project_info['projectId'])
+    os.environ[constants.ENV_VARIABLES.HOPSWORKS_PROJECT_ID_ENV_VAR] = project_id
 
+    Path(dbfs_folder).mkdir(parents=True, exist_ok=True)
+
+    get_credential(project_id, dbfs_folder)
+
+def setup_databricks(host, project_name, cert_folder="hops", port = 443, region_name = constants.AWS.DEFAULT_REGION,
+            secrets_store = 'parameterstore', api_key_file=None):
+    """
+    Set up the hopfs and hive connector on a detabricks cluster
+
+    Example usage:
+
+    >>> featurestore.setup_databricks("hops.site", "my_feature_store", "/dbfs/mnt/certs")
+
+    Args:
+        :host: the hostname of the Hopsworks cluster
+        :project_name: the name of the project hosting the feature store to be used
+        :dbfs_folder: the folder in dbfs in which to store the Hopsworks certificates and the libraries to be installed when \
+        the cluster restart
+        :port: the REST port of the Hopsworks cluster
+        :region_name: The name of the AWS region in which the required secrets are stored
+        :secrets_store: The secrets storage to be used. Either secretsmanager or parameterstore.
+
+    Returns:
+        None
+    """
+    
+    dbfs_folder = "/dbfs/" + cert_folder
```

Comment:
Is `/dbfs/`  transparent to the user? Or will the user expect to pass `/dbfs/` as well?

Reply:
actually in the example you already have `dbfs` in the parameter

Reply:
The problem is that it is a mounted folder, so the path in dbfs is the path given by the cert_folder argument, but the path in the machine has to start with /dbfs/

Reply:
fixed the example